drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -280,12 +299,60 @@ def studio_submit(self, submissions, suffix=''):
         self.weight = float(submissions['weight'])
         self.item_background_color = submissions['item_background_color']
         self.item_text_color = submissions['item_text_color']
+        # Possible ValueError should be catched in _validate_submissions
+        self.max_items_per_zone = self._get_max_items(submissions)
         self.data = submissions['data']
 
         return {
             'result': 'success',
         }
 
+    @staticmethod
+    def _get_max_items(submission):
+        """
+        Parses Max items per zone value coming from editor.
+
+        Does not catch ValueError intentionally to allow validation to capture this problem
+        """
+        raw_max_submissions = submission.get('max_items_per_zone', None)
+        if not raw_max_submissions:
+            return None
+        else:
+            return int(raw_max_submissions)
+
+    def _validate_submissions(self, submissions):
+        """
+        Validates that submissions passed are correct
+        """
+        errors = []
+
+        items = submissions['data']['items']
+
+        max_submissions = 0
```

Comment:
@e-kolpakov Similar note here: `max_items` or `max_items_per_zone` would probably be a little more accurate.